src/views/Directorio.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <section class="view-directorio">
+    <div class="directory-page">
+      <h3 class="directory-page__title">Directorio telefónico oficinas públicas</h3>
+      <input type="text" placeholder="Buscar" class="directory-page-input" v-model="search" @keyup="filterDirectory">
+      <ul class="directory-page__list">
+        <li class="directory-page__item" v-for="entity in entitiesfiltered">
+          <h3 class="directory-page__item-title">{{entity.entityLabel}}</h3>
+          <a class="directory-page__phone-number" :href="'tel:' + entity.fieldTelefono">{{entity.fieldTelefono}}</a>
+          <a :href="'mailto:' + entity.fieldCorreo">{{entity.fieldCorreo}}</a>
+        </li>
+      </ul>
+    </div>
+  </section>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+
+const query = gql `query {
+  nodeQuery(limit: 9999,
+    sort: [
+      {
+        field: "created",
+        direction: DESC
+      }
+    ],
+    filter: {
+      conditions: [
+        {
+          operator: EQUAL,
+          field: "type",
+          value: ["directorio"]
+        }
+      ]
+    }) {
+      entities {
+        entityLabel
+        ... on NodeDirectorio {
+          fieldCorreo
+          fieldTelefono
+        }
+      }
+    }
+  }`;
+
+export default {
+  name: 'directory-page',
+  apollo: {
+    nodeQuery() {
+      return {
+        query: query,
+        update(data) {
+          if (data.hasOwnProperty('nodeQuery') &&
+            data.nodeQuery.hasOwnProperty('entities') &&
+            data.nodeQuery.entities instanceof Array &&
+            data.nodeQuery.entities.length > 1) {
+              this.entities = data.nodeQuery.entities;
+              this.entitiesfiltered = data.nodeQuery.entities;
+          }
+        }
+      }
+    }
+  },
+  data() {
+    return {
+      nodeQuery: {},
+      search: '',
+      entities: {},
+      entitiesfiltered: {},
+    }
+  },
+  methods: {
+    filterDirectory() {
+      this.entitiesfiltered = this.entities.filter(entity =>
+        entity.entityLabel.toLowerCase().indexOf(this.search.toLowerCase()) > -1
+      );
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+@import "../assets/scss/variables";
+
+.view-directorio {
+  padding: 20px;
+  width: 100%;
+}
+
+.directory-page {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+.directory-page__title {
+  margin: 0 0 20px 0;
+}
+
+.directory-page-input {
+  border: solid 1px #e0e0e0;
+  height: 33px;
+  width: 100%;
+  margin-bottom: 20px;
+  color: $text;
+  background: url(.././assets/images/icons/search-icon.svg) no-repeat;
+  background-position: right 10px center;
+  padding-left: 10px;
+  &::-webkit-input-placeholder{
+    font-style: italic;
+    color: #e0e0e0;
+  }
+  &::-moz-placeholder {
```

Comment:
You're not styling the standard form of this selector (`::placeholder`). See: https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder. Also, I'm pretty sure this project has autoprefixer available. That'd mean that the browser-specific versions are not needed. Please check this.